tasks/main.yml

```diff
@@ -108,6 +108,52 @@
     cron_file: letsencrypt-renew
     user: root
 
+- name: Install maintenance backend config
+  template:
+    src: haproxy-maintenance-backends.conf
```

Comment:
The actual file name in the repo is `haproxy-maintenance-backend.conf`, without the "s".